src/model/android-versioning.js

```diff
@@ -0,0 +1,30 @@
+import * as semver from 'semver';
+
+export default class AndroidVersioning {
+  static determineVersionCode(version, inputVersionCode) {
+    if (!inputVersionCode) {
+      return AndroidVersioning.versionToVersionCode(version);
+    }
+    return inputVersionCode;
+  }
+
+  static versionToVersionCode(version) {
+    const parsedVersion = semver.parse(version);
+
+    if (!parsedVersion) {
+      return 1;
```

Comment:
should we show a warning here?

Reply:
Good idea. Initially, I had it fail the build but that's a bit aggressive if someone doesn't build for android. I also thought about only running this logic when the target is android but you'll likely want any of your build to fail early if android version generation isn't gonna work. Returning `1` was a nice balance between the two, adding a warning will make it a bit clearer though.

Reply:
Good thinking. Especially for a first iteration that is perfect. If you add the warning we should be good.